frontend/src/Components/AddedItems/AddedItems.jsx

```diff
@@ -1,15 +1,15 @@
 import React, { useState, useEffect } from 'react';
-import './addeditems.css';
+import './addeditems.scss';
```

Comment:
Please create index.js for this file
This file will include export addedItems.jsx and import './addeditems.sscss'